docs/source/contributing.rst

```diff
@@ -18,28 +18,20 @@ to, please tag your PR with ``please-do-not-edit``.
 Installing mirdata for development purposes
 ###########################################
 
-To install ``mirdata`` for development purposes:
+To install Mirdata for development purposes:
 
-    * First run:
+    - First, run ``git clone https://github.com/mir-dataset-loaders/mirdata.git``
```

Comment:
Does it work without creating a fork of the repo to contribute?

Reply:
hmm yeah, maybe it is better to explicitly say that they should do a fork and then a PR, maybe. Because they won't be able to push a new branch to mirdata if they are not a contributor